pkg/proxy/proxy.go

```diff
@@ -181,6 +181,11 @@ func (p *Proxy) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// TODO review if this is the right way to do it
```

Comment:
I believe this should be addressed by the [BuildLoadTestObject](https://github.com/hellofresh/kangal/blob/c120604e26f376f132bcd0e3ec29e22c994a1f77/pkg/proxy/proxy.go#L123) method, keeping the Proxy free of specific backend implementations

Reply:
@thiagoarrais found out that if we change the TransformLoadTestSpec to the top, that would make this "if" useless.

Reply:
If BuildLoadTestObject did the compressing itself, that would also be unnecessary. But I believe that would affect every backend instead of just JMeter. Maybe we want that?